### DISCUSSION TITLE
最近有人和我一样经常出现同步异常么？求助怎么解决

### DISCUSSION THREAD

**Discussion by spencer924** [zhaohongxuan/obsidian-weread-plugin#81] [Q&A]
![image](https://user-images.githubusercontent.com/109141979/181655925-caad5854-2909-4569-9ac7-a4ba437d2a49.png)
![image](https://user-images.githubusercontent.com/109141979/181655940-6c1ea035-e656-411c-ba32-99bbb83566ca.png)

**Comment by zhaohongxuan** [comment]
设置界面重新登录

**Reply by spencer924** [reply]
重新登录账号似乎没用，需要重新安装插件。。。

**Comment by zch1996** [comment]
我也遇到同样的问题，"Failed to fetch weread notebook reviews . Please check your Cookies and try again." 请问该怎么解决？

**Comment by mogulixmo** [comment]
我选择扫码登录，就出现一个微信读书网页版，插件上没有显示登录状态
